src/main/java/org/jahia/modules/sam/core/ProbeStatusUtils.java

```diff
@@ -0,0 +1,40 @@
+package org.jahia.modules.sam.core;
+
+import org.jahia.modules.sam.ProbeStatus;
+
+/**
+ * Simple utils class to manipulate probe data
+ */
+public final class ProbeStatusUtils {
+    private ProbeStatusUtils() {
+    }
+
+    public static ProbeStatusUtils getInstance() {
+        return InstanceHolder.instance;
+    }
```

Comment:
No need for this class to be a singleton, just declare the aggregateStatus `public static`
Singleton are interesting to keep stateful data living, but here the current class is completely stateless, so it's ok to keep it `static`.